### DISCUSSION TITLE
Skill Icons custom .png

### DISCUSSION THREAD

**Discussion by PrimalCarrot** [runelite/runelite#16643] [Q&A] [answered]
I recently installed a UI plugin that had limited change to update a few buttons and the chatbox background for OSRS and i thought " if this is doable, changing the skill icons has to be doable too". Granted this is my first time coding anything other than the classic Hello World, but if anyone can help get me on track on how i can create the plugin to have a toggle on/off and change all the skill icons from the player skill menu from classic to custom or if its even possible, that'd be awesome!

**Comment by Nightfirecat** [comment, marked as answer]
The Interface Styles plugin adjusts the sprites used in the skill menu. If you use the `2010` style, you'll see the skill icons are switched.

**Reply by PrimalCarrot** [reply]
i was able to find how the 2010 and '05 styles were changed and was able to change the .png for a skill. i know i can do this and run runelite through IntelliJ but the graphics are worse than running from Runelite. i think i know what i need to do from here though, just some trial and error! huge thanks!
